app/views/sandbox/_section.html.erb

```diff
@@ -1,4 +1,4 @@
-<% if params[metric].present? %>
+<% if @selected_metrics.include?(metric.to_s) %>
```

Comment:
Selected_metrics is just an array like `[metric1, metric2, metric3]`. I don't think we can just check `params` to see if the metric is present because now the params keys are `metricX` and now the values are the metrics.

Reply:
Creating an instance variable with @selected_metrics doesn't feel right to me. In this case I would get rid of that check entirely, and update sidebar.html.erb I would do something like:

```ruby
<%= render 'metric_option', metric: params[:metric1] if params[:metric1] %>
<%= render 'metric_option', metric: params[:metric2] if params[:metric2] %>
<%= render 'metric_option', metric: params[:metric3] if params[:metric3] %>
```